worker-languagedetection-container/testcases/automated/CAF_4001/haitian_creole.yaml.testcase

```diff
@@ -30,39 +30,39 @@ expectedOutputData:
       DetectedLanguage1_Code:
         action: "replace"
         values:
-        - data: "ht"
+        - data: "unknown"
       DetectedLanguage1_ConfidencePercentage:
         action: "replace"
         values:
-        - data: "99"
+        - data: "29"
```

Comment:
Significant change here, cannot identify haitian_creole anymore. If it is not supported then remove this testcase and make sure that haitian_creole is not referred to as a supported language.